cypress/support/commands.js

```diff
@@ -23,7 +23,34 @@
 //
 // -- This will overwrite an existing command --
 // Cypress.Commands.overwrite('visit', (originalFn, url, options) => { ... })
+import faker from 'faker';
+import './commands';
+Cypress.Commands.add('login', () => {
+  cy.task('generateUser').then((user) => {
+    const { email, username, password } = user;
 
+    cy.request('POST', '/api/users', {
+      user: {
+        email,
+        username,
+        password
+      }
+    }).then(response => {
+      const user = {
+        bio: response.body.user.bio,
+        effectiveImage: 'https://static.productionready.io/images/smiley-cyrus.jpg',
+        email: response.body.user.email,
+        image: response.body.user.image,
+        token: response.body.user.token,
+        username: response.body.user.username
+      };
+      window.localStorage.setItem('user', JSON.stringify(user));
+      cy.setCookie('auth', response.body.user.token);
+    });
+  });
+});
+
```

Comment:
The commented out block of code should be removed if it's no longer required. It encourages code cleanliness.